src/components/views/BlackCardSelection.js

```diff
@@ -6,6 +6,20 @@ import {useHistory, useLocation} from "react-router-dom";
 import BaseContainer from "../ui/BaseContainer";
 import PropTypes from "prop-types";
 
+function msToHM( ms ) {
+    // convert milliseconds to seonds:
+    let secs = ms / 1000;
+    // get hours
+    const hours = parseInt( secs/3600);
+    // secs left
+    secs = secs % 3600;
+    // get minutes
+    const minutes = parseInt( secs/60);
+    return( hours+"h:"+minutes+"min");
```

Comment:
Can you zero-pad the hours and the minutes?

Reply:
done